pkg/test/ginkgo/synthentic_tests.go

```diff
@@ -8,6 +8,12 @@ import (
 	"time"
 
 	"github.com/openshift/origin/pkg/monitor"
+	"github.com/openshift/origin/test/extended/util/disruption"
+)
+
+const (
+	// Max. duration of API server unreachability, in percents of total test duration.
+	tolerateDisruptionPercent = 1.0
```

Comment:
1.0 here mean, "never fail yet", right?  Can I get a comment with a TODO to bring us inline with upgrade tests at 0.08?

Reply:
percent != percent. In my code, it's really 1% of time it can fail. It upgrade tests, 0.08 means 8%. I can fix to 0.01, but calling something as percentage and using fraction instead is very confusing.

Reply:
changed to 0.01